### ISSUE TITLE
Fire dice & stinger dice: message incorrect

### ISSUE THREAD

**Issue by dwvanstone** [buttonmen-dev/buttonmen#1716]
If I use a Fire die and a stinger die is part of a skill attack the message is incorrect. Here's an example from the production site:
![screen shot 2015-09-20 at 9 33 13 am](https://cloud.githubusercontent.com/assets/6751265/9980955/eae71f52-5f7a-11e5-92b4-047f708e92f3.png)

It says "Turn down Fire dice by a total of 2 ..."  However since a stinger die is involved in the attack I can turn down the Fire dice anywhere from 2 through 5 points. 

The code did allow me to turn it down by 4 points so I believe the back end correctly allows it; I think only the message is incorrect.

**Comment by dwvanstone** [comment]
I have also verified on the dev site that you aren't asked if you would like to overshoot. [Not sure if this is a separate bug.]

From game 1847:
> Devious performed Skill attack using [f(8):1,g(10):8] against [sF(20):9]; Defender sF(20) was captured; Attacker f(8) rerolled 1 => 7; Attacker g(10) rerolled 8 => 8

When I performed this Skill Attack I was not asked if I wanted to overshoot. In this case, I had a shadow Fire die (showing 4) that I could have lowered in order to raise my f(8).

**Comment by jimmosk** [comment]
Let me know if I should list this as a separate issue, but I encountered a different incorrect message when using both Fire and Stinger dice, in game 1847. Here's a cap:

![screen shot 2015-09-27 at 7 00 58 pm](https://cloud.githubusercontent.com/assets/6092544/10125676/6158c84a-654b-11e5-9799-5536df6548ca.jpg)

I did a skill attack including a stinger die on a die showing 14. My dice added up to anywhere between 7 and 9, so I'd need to turn my Fire die down anywhere between 5 and 7. That cap shows what happened when I tried to turn it down by 8 instead: it didn't allow the attack to happen, but didn't say why not, nor even that there was a problem; the message displayed was simply "Loaded data for game 1847."
